src/drc/datamodel/models.py

```diff
@@ -356,3 +366,33 @@ def unique_representation(self):
             io_id = request_object_attribute(self.object, 'identificatie', self.object_type)
             self._unique_representation = f"({self.informatieobject.latest_version.unique_representation()}) - {io_id}"
         return self._unique_representation
+
+
+class BestandsDeel(models.Model):
+    uuid = models.UUIDField(
+        unique=True, default=_uuid.uuid4,
+        help_text="Unieke resource identifier (UUID4)"
+    )
+    informatieobject = models.ForeignKey(
+        'EnkelvoudigInformatieObjectCanonical', on_delete=models.CASCADE, related_name='bestandsdelen'
+    )
+    index = models.PositiveIntegerField(
+        help_text=_("Een volgnummer dat de volgorde van de bestandsdelen aangeeft.")
+    )
+    grootte = models.PositiveIntegerField(
+        help_text=_("De grootte van dit specifieke bestandsdeel.")
+    )
+    inhoud = PrivateMediaFileField(
+        upload_to='uploads/%Y/%m/', blank=True,
+        help_text=_("De (binaire) bestandsinhoud van dit specifieke bestandsdeel.")
+    )
+
+    class Meta:
+        unique_together = ('informatieobject', 'index')
```

Comment:
can you set `verbose_name` and `verbose_name_plural` as well, so that it looks good in the admin? And also add the model to the admin with some convenient `list_filter` etc. Those that debug 'live' data will thank you :smile: